.travis.yml

```diff
@@ -51,13 +51,6 @@ jobs:
     env: BUILD_WHEELS=1
     script: tools/wheels/build-wheels.sh ${LIBRDKAFKA_VERSION#v} wheelhouse
 
-  - name: "Wheels: MacOSX x64"
```

Comment:
Why is this removed?

Reply:
These wheels are generated with Semaphore as well. Travis requires credits OS wheel generation.